rpm/0004-Fix-busybox-compatibility-for-test-execute.patch

```diff
@@ -0,0 +1,43 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Tomi=20Lepp=C3=A4nen?= <tomi.leppanen@jolla.com>
+Date: Wed, 9 Mar 2022 14:14:53 +0200
+Subject: [PATCH] Fix busybox compatibility for test-execute
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Tomi Leppänen <tomi.leppanen@jolla.com>
+---
+ test/test-execute/exec-specifier.service  | 2 +-
+ test/test-execute/exec-specifier@.service | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/test/test-execute/exec-specifier.service b/test/test-execute/exec-specifier.service
+index 37852390ac..3e0074ec10 100644
+--- a/test/test-execute/exec-specifier.service
++++ b/test/test-execute/exec-specifier.service
+@@ -14,7 +14,7 @@ ExecStart=/usr/bin/test %t = /run
+ ExecStart=/usr/bin/test %S = /var/lib
+ ExecStart=/usr/bin/test %C = /var/cache
+ ExecStart=/usr/bin/test %L = /var/log
+-ExecStart=/bin/sh -c 'test %u = $$(id -un 0)'
++ExecStart=/bin/sh -c 'test %u = root'
```

Comment:
If the point is just to ensure this gets executed as root while avoiding literal "root": I guess this could be 'test %U = 0' too?

Reply:
That %U = 0 would be the test just below.

Reply:
Now it uses `getent` so it should be fine.

Reply:
:+1: